zerver/webhooks/teamcity/doc.md

```diff
@@ -1,42 +1,51 @@
+# Zulip TeamCity integration
+
 Get Zulip notifications for your TeamCity builds!
 
+{start_tabs}
+
 1. {!create-channel.md!}
 
 1. {!create-an-incoming-webhook.md!}
 
-1. {!generate-integration-url.md!}
+1. {!generate-webhook-url-basic.md!}
 
-1. Install the [tcWebHooks plugin](https://github.com/tcplugins/tcWebHooks/releases)
-   onto your TeamCity server. Follow the plugin instructions in your
-   TeamCity documentation, or refer to [the online TeamCity documentation][1].
+1. Install the [tcWebHooks plugin][1] onto your TeamCity server. Follow the
+   plugin instructions in your TeamCity documentation, or refer to
+   [the online TeamCity documentation][2].
 
 1. Go to your TeamCity **Overview** page. Select the **Project** or **Build**
    you'd like to receive notifications about, and click on the **WebHooks** tab.
-   If you'd like to configure webhooks for a **Project**, click on
-   **Add project WebHooks**. If you'd like to configure webhooks for a specific
-   **Build**, click on **Add build WebHooks**. Click on
-   **Click to create new WebHook for this project/build**.
+   Click **Add project WebHooks** for a **Project**, or click **Add build WebHooks** for a **Build**. Select **Click to create new WebHook for this project/build**.
 
-1. Set **URL** to the URL constructed above. Set **Payload Format** to
+1. Set **URL** to the URL generated above, set **Payload Format** to
    **Legacy Webhook (JSON)**. Uncheck all **Trigger on Events** options,
    and check **Trigger when build is Successful** and **Trigger when build Fails**.
+   Click **Save**.
 
-1. Optionally, check **Only trigger when build changes from Failure to Success**
-   and **Only trigger when build changes from Success to Failure**.
-
-1. Click **Save**.
+    !!! tip ""
+        You may instead choose the options
+        **Only trigger when build changes from Failure to Success** or
+        **Only trigger when build changes from Success toFailure**, as suitable.
 
-[1]: https://confluence.jetbrains.com/display/TCD9/Installing+Additional+Plugins
```

Comment:
The reference links were previously used at the bottom of each section, moved them to the bottom of the document instead.

Reply:
Looking at the screenshots in their documentation, this would probably be better in the instruction.